clang/unittests/Sema/HeuristicResolverTest.cpp

```diff
@@ -41,7 +61,16 @@ template <typename InputNode, typename ParamT, typename InputMatcher,
           typename... OutputMatchers>
 void expectResolution(llvm::StringRef Code, ResolveFnT<ParamT> ResolveFn,
                       const InputMatcher &IM, const OutputMatchers &...OMS) {
```

Comment:
We can't use a trailing default parameter here, so thought of adding an `IgnoredDiagnostics` to the function. 

```suggestion
void expectResolution(llvm::StringRef Code, 
                      llvm::SmallSet<uint, 16> IgnoredDiagnostics, 
                      ResolveFnT<ParamT> ResolveFn,
                      const InputMatcher &IM, const OutputMatchers &...OMS) {
```